public/sitemap-0.xml

```diff
@@ -1,746 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
-  xmlns:news="http://www.google.com/schemas/sitemap-news/0.9"
-  xmlns:xhtml="http://www.w3.org/1999/xhtml"
-  xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0"
-  xmlns:image="http://www.google.com/schemas/sitemap-image/1.1"
-  xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
-  <url>
-    <loc>https://juzibot.com</loc>
-    <lastmod>2023-08-29T07:08:15.151Z</lastmod>
-    <changefreq>daily</changefreq>
-    <priority>1</priority>
-  </url>
```

Comment:
What is the reason that clear this sitemap?

Reply:
I'm not sure why that happened. I reverted the changes